src/keyword-extractor.py

```diff
@@ -34,7 +36,7 @@
 os.chdir(os.path.dirname(os.path.realpath(__file__)))
 
 with open(PLANS_FILE) as json_file:
-    plans_dict = json.load(json_file)
+    plans = json.load(json_file)
```

Comment:
These are the only real changes to this file. The rest are just `black` autoformatting the file